spec/controllers/callbacks_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe CallbacksController, type: :controller do
+  before do
+    @request.env['devise.mapping'] = Devise.mappings[:user]
+    @request.env['omniauth.auth'] =
+      OmniAuth.config.mock_auth[:github] =
+        OmniAuth::AuthHash
+        .new({ 'extra' => OmniAuth::AuthHash.new({
```

Comment:
@buys-fran Not sure if we are using standardrb or rubocop, but is this the code style we want in the app?

Reply:
Thanks for nitpicking. I realised my editor setup was not using standardrb to fix the code.
Now that I got it right, the code looks much better.